packages/cozy-procedures/src/redux/amountSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from 'redux-starter-kit'
+
+const amountSlice = createSlice({
+  initialState: 20000,
+  slice: 'amount',
+  reducers: {
+    update: (state, action) => {
+      return action.payload.value
+    }
+  }
+})
+
+const { actions, reducer } = amountSlice
```

Comment:
do we have `actions`?

Reply:
Yes, there's the `update` action. Unless I misunderstood the question?